### PR TITLE
first version of blink

### PR DESCRIPTION
**Pull request by msqintian**

### CODE REVIEW

File: blink.js [hunk 1]
```diff
@@ -1 +1,17 @@
-// YOUR CODE GOES HERE
+(function($){
+  $.fn.blink = function(speed){
+    var that = this;
```
Comment (afeld) [inline blink.js:11]: As long as you're going to make an alias of the variable, you might as well make it something more descriptive like `$el`.

File: index.html [hunk 2]
```diff
@@ -10,7 +11,7 @@
       I'm a <code>blink</code> tag!
     </div>
     <script>
-      jQuery('.myDiv').blink(1000);
+      jQuery('.myDiv').blink('1000');
```
Comment (afeld) [inline index.html:14]: Why the change?

**Comment by afeld** [comment]
Can you try re-working your solution using a single `setTimeout()` or `setInterval`?  Thanks!

**Comment by msqintian** [comment]
add code to only use setTimeout to implement the blink function

**Comment by afeld** [comment]
Well done!  You might check out some of [the other solutions](https://github.com/advanced-js/blink/pulls?state=closed) to see how this can be done more succinctly.
